internal/pwru/xdp_metadata.go

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Maybe we could make skb_metadata.go generic, so that it can be reused by XDP?

Reply:
Sure. I will do it.